AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -29,6 +30,36 @@
 
 NSInteger const ASDefaultDrawingPriority = ASDefaultTransactionPriority;
 
+@interface _ASDisplayNodePosition : NSObject
```

Comment:
If we need this in the final version, will likely want to switch to structs + vectors.